src/components/notification-detail.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import { Card, CardContent, Stack, FormLabel } from '@mui/material';
+import { useTranslator } from '../hooks';
+import { Scheduler } from '../handler';
+import { LabeledValue } from '../components/labeled-value';
+
+interface INotificationItemProps {
+  label: string;
+  value: string | boolean;
+}
+
+interface INotificationDetailsProps {
+  notification: Scheduler.INotification;
+}
```

Comment:
nit: [use `type` instead of `interface`](https://www.totaltypescript.com/type-vs-interface-which-should-you-use#default-to-type-not-interface). Interfaces are just more characters to type and have some surprising side-effects like declaration merging.

This is personal preference; change is encouraged but not necessary.

Reply:
Thanks for the link. I'm actually defaulting to interfaces partly based on typescript handbook [recommendation](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces) that article you link references and disagrees with.